src/riak_kv_ts_util.erl

```diff
@@ -112,25 +112,30 @@ build_sql_record_int(describe, SQL, _Cover) ->
     {ok, #riak_sql_describe_v1{'DESCRIBE' = D}};
 
 build_sql_record_int(insert, SQL, _Cover) ->
-    T = proplists:get_value(table, SQL),
-    case is_binary(T) of
+    Table = proplists:get_value(table, SQL),
```

Comment:
Verbosity for the win! Never been a fan of single letter variables. Except when I was coding AppleSoft BASIC which *only* had single character variables.